src/project_template/main.py

```diff
@@ -1,10 +1,13 @@
-from project_template import config
-from project_template.utils.log import log
+import config
+from utils.log import log
 
 
+# TODO figure out why the package is working some times and not others `import project_template....`
```

Comment:
Did you figure this out? Do we want to merge with this still being a problem?

Reply:
Fixed.